src/main.cpp

```diff
@@ -40,24 +41,24 @@ bool parseOptions(int argc, char** argv, openblack::Arguments& args, int& return
 	{
 		loggingSubsystems += std::string(", ") + system.data();
 	}
+	const auto config = StartupConfig(std::filesystem::current_path() / "config.json");
```

Comment:
Current path is a good idea, I also think we should store something globally for the user such as in the user config directory

Function name | Value on Linux | Value on Windows | Value on macOS
-- | -- | -- | --
config_dir | `$XDG_CONFIG_HOME`       or `$HOME/.config` | `{FOLDERID_RoamingAppData}` or `{FOLDERID_LocalAppData}` | `$HOME/Library/Application Support`

There's also android to consider.